packages/clerk-js/src/ui/components/UserProfile/DeletePage.tsx

```diff
@@ -29,6 +29,7 @@ export const DeletePage = withCardStateProvider(() => {
     type: 'text',
     label: localizationKeys('formFieldLabel__confirmDeletion'),
     isRequired: true,
+    // TODO: localize this
```

Comment:
We should add a ticket for this, or else it will be forgotten.

Reply:
it's a minor thing, i'll add this with this PR.